### PR TITLE
refactor: late october edition

### LINKED ISSUE #130: Add rate limit to prevent spamming
Some people are spamming `/joke`

### PR DESCRIPTION
**Pull request by aldy505**
a lot of stuff going on

closes #130

### CODE REVIEW

**Review by ronnygunawan — APPROVED** [review]

File: src/services/blog/index.js [hunk 1]
```diff
@@ -1,11 +1,11 @@
-import { getTheDevRead } from './request.js';
-import redisClient from '#utils/redis.js';
-import { randomArray } from '#utils/random.js';
-import { renderTemplate } from '#utils/template.js';
-import { getCommandArgs } from '#utils/command.js';
-import { logger } from '#utils/logger/logtail.js';
+import { getTheDevRead } from "./request.js";
```
Comment (elianiva) [inline src/services/blog/index.js:1]: file ini direname jadi `devread` aja deh, bikin bingung kalo `blog`

File: src/utils/redis.js [hunk 2]
```diff
@@ -1,6 +1,6 @@
 /**
  * @file redis.js
- * @fileoverview Async Redis functions with JSDoc annotations
+ * @fileoverview Redis functions with JSDoc annotations
```
Comment (elianiva) [inline src/utils/redis.js:3]:
```suggestion
 * @fileoverview Redis functions wrapped in Promise API with JSDoc annotations
```

File: src/services/blog/index.js (hunk 1 above)
Comment (aldy505) [reply src/services/blog/index.js:1]: ok later this midnight ya

File: src/utils/redis.js (hunk 2 above)
Comment (aldy505) [reply src/utils/redis.js]: gapenting woi wkwkkw

**Review by artileda — APPROVED** [review]

**Pull request merged**